src/test/java/org/broadinstitute/consent/http/db/DatasetDAOTest.java

```diff
@@ -372,28 +403,12 @@ public void testGetDictionaryTerms() {
             assertNotNull(t.getKey());
         });
     }
-    @Test
-    public void testGetDataSetsForObjectIdList() {
```

Comment:
Can you explain why some of these tests are no longer relevant? Is it because it was testing the duplicated codepaths / queries you mentioned? Also, what makes you confident that these should be deleted?